torchtune/models/llama2/_model_builders.py

```diff
@@ -369,6 +368,83 @@ def lora_llama2_reward_7b(
         quantize_base=quantize_base,
     )
 
+# Add llama multi classification model builders
```

Comment:
I definitely want to make it easier to use classification models in our codebase, this is high-up on my TODO list - your comment about replacing reward models is on-point. Soon you should just be able to use wrap any existing model builder without having to re-define builders.